x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -505,6 +511,40 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
             rowHeightsOptions={props.rowHeightsOptions}
             onColumnResize={onColumnResize}
             ref={dataGridRef}
+            renderCustomGridBody={
+              props.dynamicRowHeight
+                ? ({ visibleColumns: _visibleColumns, Cell }) => {
+                    return (
+                      <>
+                        {oldAlertsData
+                          .concat(
+                            isLoading
+                              ? Array.from({ length: pagination.pageSize - oldAlertsData.length })
+                              : []
+                          )
+                          .map((_row, rowIndex) => (
+                            <Row
+                              role="row"
+                              className={`euiDataGridRow ${
+                                rowIndex % 2 !== 0 ? 'euiDataGridRow--striped' : ''
```

Comment:
[Optional heads up] If/when EUI converts EuiDataGrid from Sass to Emotion/CSS-in-JS*, this className will stop working. You may want to consider baking this logic into your `styled.div` now, if that's an option - the color being used is `euiColorLightestShade`.

*Note: if another team ends up taking ownership of EuiDataGrid and it moves out of EUI, this migration may not end up happening, so totally up to you all as if you'd rather address this later.